packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -219,9 +216,7 @@ export class L2Bridge {
     return this.walletAddressCache
   }
 
-  public async getAndUpdateL2EthBalance(): Promise<BigNumber> {
```

Comment:
another rider - stop storing balance as a class variable; useless (client can/will probably store it anyway) and confusing